### DISCUSSION TITLE
Setting save options by code

### DISCUSSION THREAD

**Discussion by vinRosso** [Autodesk/maya-usd#3657] [Q&A] [answered]
How do I set by code the saving option to `Save all edits (including USD) to the Maya scene file`?

I can easily avoid the window to pop-up with `cmds.optionVar(intValue=["mayaUsd_SerializedUsdEditsLocationPrompt", 0])`, but I couldn't find a way to set the actual option.

**Comment by seando-adsk** [comment, marked as answer]
That would be the optionVar "mayaUsd_SerializedUsdEditsLocation", which takes an int:
- 1 = save usd files separately
- 2 = save usd to the maya scene file
- 3 = save maya scene file only, usd will not be saved

**Reply by vinRosso** [reply]
Wonderful, thanks!

**Reply by subing85** [reply]
Is it possible to manage when  "save usd files separately" USD File Extensions  (args={"format": "usda"} )

**Reply by seando-adsk** [reply]
Yes that would be the optionVar "mayaUsd_SaveLayerFormatArgBinaryOption" which is `0` for Ascii and `1` for binary.

Sean
